include/vrv/scoredefinterface.h

```diff
@@ -33,6 +33,7 @@ class ScoreDefInterface : public Interface,
                           public AttLyricStyle,
                           public AttMeasureNumbers,
                           public AttMidiTempo,
+                          public AttMmTempo,
```

Comment:
You need to also add it to `scoredefinterface.cpp` and add the reset method there.

Reply:
Thanks a lot @rettinghaus! I have added that in c67e664 below.